arrayAnalysis.js

```diff
@@ -0,0 +1,19 @@
+const arrayAnalysis = (numbers) => {
+  const average = numbers.reduce((acc, cr) => {
+    // eslint-disable-next-line no-param-reassign
+    acc += cr;
+    return acc;
+  }) / numbers.length;
```

Comment:
You can return the addition of the accumulator plus the current value, reduce is going to accumulate for you:

```js
const average = numbers.reduce((acc, cr) => acc + cr, 0) / numbers.length;
```

Use a `0` for the initial value in case you are given an empty array.